templates/lxc-opensuse.in

```diff
@@ -138,7 +138,11 @@ download_opensuse()
     # download a mini opensuse into a cache
     echo "Downloading opensuse minimal ..."
     mkdir -p "$cache/partial-$arch-packages"
-    zypper --quiet --root $cache/partial-$arch-packages --non-interactive ar http://download.opensuse.org/distribution/$DISTRO/repo/oss/ repo-oss || return 1
+    if [[ $DISTRO == "tumbleweed"
```

Comment:
haha, yeah, that's certainly bad. The partial line above is also a bashism.

In sh, you'd want:
```
if [ "${DISTRO}" = "tumbleweed" ]; then
```